src/rest-server/src/templates/dockerContainerScript.mustache

```diff
@@ -137,7 +137,7 @@ function start_ssh_service()
   printf "%s %s\n" \
     "[INFO]" "start ssh service"
   cat /root/.ssh/{{ jobData.jobName }}.pub >> /root/.ssh/authorized_keys
-  sed -i 's/Port.*/Port '$PAI_CONTAINER_SSH_PORT'/' /etc/ssh/sshd_config
+  sed -i 's/.Port.*/Port '$PAI_CONTAINER_SSH_PORT'/' /etc/ssh/sshd_config
```

Comment:
need to work with the old `Port 22` line
```suggestion
  sed -i 's/[# ]*Port.*/Port '$PAI_CONTAINER_SSH_PORT'/' /etc/ssh/sshd_config
```

Reply:
it is better than my commit, thanks!

Reply:
could you help to update your PR branch?

Reply:
OK, thanks!